modules/virt-template-fields-for-boot-source.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+[id="virt-template-fields-for-boot-source_{context}"]
+= Virtual machine template fields for adding a boot source
+
+The following table describes the fields for *Add boot source to template* dialog. This dialog displays when you click *Add Source* for a virtual machine template in the *Templates* tab.
+[cols="2a,3a,5a"]
+|===
+|Name | Parameter |  Description
+
+.4+|Boot source type
+
+|Upload local file (creates PVC)
+|Upload a file from your local device. Supported file types include gz, xz, tar, and qcow2.
+
+|Import via URL (creates PVC)
+|Import content from an image available from an HTTP or S3 endpoint. Obtain the download link URL from the web page where the image download is available and enter that URL link in the *Import via URL (creates PVC)* field. Example: For a Red Hat Enterprise Linux image, log on to the Red Hat customer portal, access the image download page, and copy the download link URL for the KVM guest image.
+
+|Clone existing PVC (creates PVC)
+|Use a PVC that is already available in the cluster and clone it.
+
+|Import via Registry (creates PVC)
+|Specify the bootable operating system container that is located in a registry and accessible from the cluster. Example: kubevirt/cirros-registry-dis-demo.
+
+|Source provider
+|
+|Optional field. Add descriptive text about the source for the template or the name of the user who created the template. Example: Red Hat.
+
+.3+|Advanced
+|Storage class
+|The storage class that is used to create the disk.
+
+|Access mode
+|Access mode of the persistent volume. Supported access modes are *Single User (RWO)*, *Shared Access (RWX)*, *Read Only (ROX)*. If *Single User (RWO)* is selected, the disk can be mounted as read/write by a single node. If *Shared Access (RWX)* is selected, the disk can be mounted as read-only by many nodes. The `kubevirt-storage-class-defaults` config map provides access mode defaults for data volumes. The default value is set according to the best option for each storage class in the cluster.
```

Comment:
`If *Shared Access (RWX)* is selected, the disk can be mounted as read-only by many nodes`.

is `read-only` correct in this sentence? I think it's read/write just like what is written in `If *Single User (RWO)* is selected, the disk can be mounted as read/write by a single node.`

Reply:
Corrected the text.